docs/Plugins-evals.md

```diff
@@ -0,0 +1,241 @@
+
+##### 2.G.2 – Account Discovery (T1087) via Command-Line Interface (T1059)
+The `net` utility is executed via `cmd` to enumerate detailed information about a specific user account.
+
+##### 2.H.1 – Query Registry (T1012) via Command-Line Interface (T1059)
+The `reg` utility is executed via `cmd` to enumerate a specific Registry key associated with local system policies to ensure that the user will not be prompted for credentials when elevating permissions.
+
+#### Step 3 - Privilege Escalation
+
+Step 3 tested the following techniques:
+
+##### 3.A.1 - Bypass User Account Control (T1088) via Access Token Manipulation (T1134)
+This Eval plugin ability uses [Empire's Invoke-BypassUACTokenManipulation.ps1](https://github.com/EmpireProject/Empire/blob/master/data/module_source/privesc/Invoke-BypassUACTokenManipulation.ps1) script to steal the token of an existing high-integrity process and launch a new, high-integrity CALDERA RAT with limited functionality.
+
+\*Note if a high integrity process is not running the script will attempt to spawn TaskMgr.exe with the RunAs flag. If there is no high integrity process running or TaskMgr.exe fails to spawn as admin the script will fail. We recommend spawning cmd.exe as the high integrity process prior to starting the DIY Eval. Read more about the lab setup [here]()
+
+### Adversary Profile 2 - ATTACK Eval APT 3 Scenario 1 Step 3B-3C
+
+#### Step 3 Continued - Privilege Escalation
+
+The Evals plugin emulates privilege escalation by elevating its' RAT’s process integrity level from medium to high, while maintaining the current user. Step 3 began in the first Adversary Profile with the execution of an Empire Bypass UAC technique. The Bypass UAC technique resulted in a high integrity process that had limited functionality, but allows CALDERA to inject into an already existing high-integrity process, which is not limited in functionality.
+
+Step 3 tested the following techniques:
+
+##### 3.B.1 - Process Discovery (T1057) and 3.C.1 - Process Injection (T1055)
+The limited functionality high-integrity RAT injects malicious base64 encoded PowerShell into an existing fully functional high-integrity process, resulting in a new elevated, fully functional high-integrity RAT.
+
+This Eval plugin ability uses [Empire's Invoke-PSInject](https://github.com/EmpireProject/PSInject/blob/master/Invoke-PSInject.ps1) to perform the injection. This  ability added the [EnumProcesses function from Empire's Invoke-BypassUACTokenManipulation.ps1](https://github.com/EmpireProject/Empire/blob/master/data/module_source/privesc/Invoke-BypassUACTokenManipulation.ps1#L876) script in, as it now allows CALDERA to find a high integrity process to inject in to on the fly.
+
+\*Note like Step 3.A if there is no high integrity process running and owned by the current user the script will fail. We recommend spawning cmd.exe as the high integrity process prior to starting the DIY Eval. Read more about the lab setup [here]()
```

Comment:
Not sure where this link was meant to point to.

Reply:
https://attackevals.mitre.org/methodology/round1/environment.html